packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -4,19 +4,16 @@ import {__RouterContext} from "react-router";
 
 import type {AriaProps} from "@khanacademy/wonder-blocks-core";
 
-import Color, {mix, fade} from "@khanacademy/wonder-blocks-color";
```

Comment:
**praise:** Neat! thanks for moving everything from `Color, Spacing` to `tokens` 👏